test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_send_HB.lua

```diff
@@ -0,0 +1,98 @@
+--  Requirement summary:
+--  [Services]: SDL must be able to respond to Heartbeat_request over control service from mobile app
+--  [HeartBeat][Genivi]: SDL must track sending of HeartBeat_request from/to mobile app
+--
+--  Description:
+--  Check that no heartbeat occurs if App uses v3 protocol version and send HB to SDL in time less than HB timeout.
```

Comment:
@LitvinenkoIra, may be will be better if you write heartbeat timeout, instead only heartbeat.